packages/edit-site/src/components/template-switcher/style.scss

```diff
@@ -6,6 +6,13 @@
 	position: relative;
 }
 
+.edit-site-template-switcher__label-home-icon {
+	width: 24px;
+	height: 24px;
+	position: absolute;
```

Comment:
Definitely not a blocker, but you could use something like [this demo](https://codepen.io/MichaelArestad/pen/7c7db98a6bd859d2a5fdd45f82650a39) to make it a little more flexible for adding in things like the dot icon.

Reply:
Yes, we should probably switch to this soon. I thought I had done it for https://github.com/WordPress/gutenberg/pull/21202 but it looks like I never did it.